docs/extensions/logging-and-monitoring.md

```diff
@@ -90,23 +90,27 @@ data:
 
 ## Logging
 
-The central fluentd/fluent-bit instances in the `garden` namespace are parsing the logs from all containers in the seed cluster.
+The central fluent-bit instances in the `garden` namespace are parsing the logs from all containers in the seed cluster.
 The shoot-specific instances only extract exactly those logs for the pods of the control plane they are responsible for.
 This allows to only fetch the logs for the pods once for the whole cluster, and to distribute them afterwards.
 
+All of the components have custom filters and parsers defined for parsing their log messages correctly. Currently in the extensions we create configmap under /charts directory called `logging-config` where their filters and parsers are. The fluent-bit reads this type of config files.
+
 Extension controllers might deploy components as part of their reconciliation next to the shoot's control plane.
 Examples for this would be a cloud-controller-manager or CSI controller deployments.
-In some cases, the extensions want to submit logging parse configuration for these components such that their logs can be scraped by Gardener's fluentd/fluent-bit deployment(s), and later be visible in the Kibana dashboards.
+In some cases, the extensions want to submit logging parse configuration for these components such that their logs can be scraped by Gardener's fluent-bit deamonset(s), and later be visible in the Grafana dashboards.
```

Comment:
So you have understand it well. For each new extension controller one could add labeled ConfigMap in order to make logs visualization more verbose in the Grafana.
For components that are always there(by default) we use the direct config map in the well-know location.
But for extensions we use config map. Why? I don't remember but the current fluent-bit config map is a way too big and adding parsers and filters for the extensions will make it even bigger. On the other hand if someone wants to write his own extension controller is not obligated to make PR to the Gardener repository.

Reply:
Shortly, this is part of the extensibility contract.
It is needed so that each provided can configure the logging sertings for the components that it deploys in the seed like `cloud-controller-manager`, `aws-lb-readvertiser` and others.

More details here https://github.com/gardener/gardener/issues/1351